### PR TITLE
CPLP-1806 extend services ep

### PR DESCRIPTION
**Pull request by Phil91**
In this PR [CPLP-1804](https://jira.catena-x.net/browse/CPLP-1804) & [CPLP-1806](https://jira.catena-x.net/browse/CPLP-1806) are resolved.

### CODE REVIEW

**Comment by ntruchsess** [comment]
secret-service-appsettings.json lacks SalesManagerRoles in ServiceSettings to run, please update the secrets

**Review by ntruchsess — APPROVED** [review]

**Comment by ntruchsess** [comment]
Id did some cleanup and refactoring. Please double-check before merging. (Don't forget to update the secrets first!)

File: src/portalbackend/PortalBackend.DBAccess/Repositories/OfferRepository.cs [hunk 1]
```diff
@@ -240,18 +249,35 @@ public OfferDescription AttachAndModifyOfferDescription(Guid offerId, string lan
...
+         Pagination.CreateSourceQueryAsync(
+                 skip,
+                 take,
+                 _context.Offers
+                     .AsNoTracking()
+                     .Where(x => 
+                         x.OfferTypeId == OfferTypeId.SERVICE &&
+                         x.OfferStatusId == OfferStatusId.ACTIVE &&
+                         (serviceTypeId == null || x.ServiceTypes.Any(st => st.Id == serviceTypeId)))
+                     .GroupBy(s => s.Id),
```
[the remark this answers is not in the document]
Comment (Phil91) [reply src/portalbackend/PortalBackend.DBAccess/Repositories/OfferRepository.cs]: 👍 thanks for the explination and the fix

**Comment by Phil91** [comment]
> I did some cleanup and refactoring. Please double-check before merging. (Don't forget to update the secrets first!)

secrets are updated and uploaded

**Pull request merged**